pyhole/core/utils.py

```diff
@@ -289,10 +292,37 @@ def generate_config():
     print "Done"
 
 
+def datetime_now_string():
+    return datetime.datetime.utcnow().isoformat()
+
+
 def fetch_url(url, **kwargs):
-    """Fetch a URL."""
```

Comment:
Good move here. This additional request functions are helpful to have. Could you do a `git grep fetch_url` to find all references to `fetch_url()` and change them to `get_url`? We can then do away with this improperly-named function.

I'm also thinking it might be more ideal to turn these request helpers into a class, so we don't have to have all the duplicate session information. Thoughts? It could be broken out into its own core module (`pyhole.core.url`).